lib/org/chromium/webidl/ast/Argument.js

```diff
@@ -24,6 +24,11 @@ foam.CLASS({
       class: 'Boolean',
       name: 'isVariadic',
     },
+    {
+      class: 'String',
+      name: 'id',
+      factory: function() { return this.getName(); },
```

Comment:
This and all the AST changes are in an attempt to give unnamed items names.
Not only does this speed up the diff (since an adapt on array of members sorts the items), but it also makes diff more straightforward, as we can directly look at the name.